bundle/manifests/controller-manager-metrics-service_v1_service.yaml

```diff
@@ -3,7 +3,7 @@ kind: Service
 metadata:
   creationTimestamp: null
   labels:
-    control-plane: controller-manager
+    app.kubernetes.io/name: lvm-operator
```

Comment:
@iamniting I'm seeing the labels getting updated after running the `make bundle`.  Do you know if this is expected?

Reply:
yes this is expected